doc/internal/man3/DEFINE_LIST_OF.pod

```diff
@@ -56,6 +59,12 @@ B<OSSL_LIST_MEMBER>(B<I<NAME>>, B<I<TYPE>>) field.
 B<ossl_list_I<NAME>_init>() initialises the memory pointed to by I<list>
 to zero which creates an empty list.
 
+B<ossl_list_I<NAME>_init_elem>() initialises the list related memory pointed
```

Comment:
Shouldn't more or less all the `I<NAME>` be `I<TYPE>`?

Reply:
Preexisting but fixed.  Dunno how I didn't notice and that it got through review previously :(